### PR TITLE
Legend and Editor widget in an accordion pane

### PR DESCRIPTION
**Pull request by marlak**
better version of #32 with better formatting

This sample shows how to incorporate the editor widget in one of the
legend's accordion panes

### CODE REVIEW

File: web-js/legend-Editor/legendEditor.html [hunk 1]
```diff
@@ -0,0 +1,239 @@
...
+	  height:99%;
+	  width:99%;
+	  }
+	  
+      #map{
+        padding:0;
+      }
+     
+	  
+	  #leftPane{
```
Comment (jgravois) [inline web-js/legend-Editor/legendEditor.html]: css is starting to drift here

File: web-js/legend-Editor/legendEditor.html [hunk 2]
```diff
@@ -0,0 +1,239 @@
...
+        "esri/layers/FeatureLayer",
+
+        "esri/symbols/SimpleMarkerSymbol",
+        "esri/symbols/SimpleLineSymbol",
+
+        "esri/dijit/editing/Editor",
+        "esri/dijit/editing/TemplatePicker",
+		
+		
+		"esri/dijit/Legend",
```
Comment (jgravois) [inline web-js/legend-Editor/legendEditor.html]: drifted AMD module

**Comment by jgravois** [comment]
nice work marla, but there's no need to close PRs and open a new one to fix minor issues.  just lay down additional commits to the same branch (in this case `legend`) locally and push up to the same branch in your fork stored on github.  afterwards the updates will appear automatically in the Pull Request (which is comparing the latest changes to your fork to the upstream master)

**Comment by nohe427** [comment]
Looks like @marlak fixed the formatting issues.  I will merge.

**Pull request merged**